tests/testsuite/alt_registry.rs

```diff
@@ -760,6 +760,11 @@ fn no_api() {
         .with_stderr_contains(&err)
         .run();
 
+    p.cargo("owner --registry alternative list")
```

Comment:
Not sure what is going on with all of these other test changes.

Let me re-phrase.  We need to make sure the old flags continue to work.  I do not think we need to exhaustively test all behavior of all flags.  What I would recommend is duplicating the code set of `cargo owner` tests (these are most likely to only exist in the `owner.rs` file) that can verify we are acknowledging all of the arguments.  Any of the tests focusing on behavior deeper down in the stack, I would just switch to the new CLI.

Reply:
I understand what you mean should be to reduce some test cases. I removed some test cases.